### PR TITLE
06. Ansible.run

### PR DESCRIPTION
**Pull request by Zhenyakuch**

### CODE REVIEW

File: Evgeniya_Pehota/06. Ansible.run/06. Ansible.run.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+PLAY [work_hosts] ******************************************************************************************************
+
+TASK [Gathering Facts] *************************************************************************************************
+Thursday 08 July 2021  21:38:40 +0000 (0:00:00.079)       0:00:00.079 *********
+ok: [work_01]
+ok: [work_02]
+
+TASK [Set authorized key taken from file] ******************************************************************************
+Thursday 08 July 2021  21:38:42 +0000 (0:00:02.195)       0:00:02.275 *********
+ok: [work_01] => (item=ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQD9eMhBCgVVDnENFCd1dz7jFtiPjSa8rqaJYuga63Ll89DZ1ynHnJ49Inn9PDwGPWIr41eFB75wWAASXJ+NaZh7Ly1gyCA+PACjZkYILf/QecoJhOs+xj4X79xvnyt5Tlj0ndMhUPHaHnSollWUzyvBJ9rXWv4/UxiVQtFNam1ridWieION4D/CovPFZg01ughdNI+CBM+2lgoaA0BIKZIpTVX9vCvJYy17u4bdX9XIGGdIMo8uE7AXEhEShgOIwowJkfQ5owgEnfxk5w4TfKZvGntLxaYtL11rYGd5JzZGZOKwgLXvwQZrs+gj0A2Uk8gh3ssA/5UKuGjjSWVjHWy9jv8uaee+mpxV2cFDT+6rYxtl/wiKRsSKYDapmuxawIAB0g3b6urlPbLnvIOJQkykvoMV7Q01HCpUKk/dOc0G+7H+u5+PXR9UWP2MQKsxsQ5/7El2B6HLzdPCC2m0XRmmFawieDm7UjP7/juD/r96USnV2pW4okXhgp5/NmHUT1k= vagrant@vagrant)
```
Comment (pluhin) [inline Evgeniya_Pehota/06. Ansible.run/06. Ansible.run.md:70]: Please remove sensitive data from PR

File: Evgeniya_Pehota/06. Ansible.run/06. Ansible.run.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+ok: [work_02] => (item=ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQD9eMhBCgVVDnENFCd1dz7jFtiPjSa8rqaJYuga63Ll89DZ1ynHnJ49Inn9PDwGPWIr41eFB75wWAASXJ+NaZh7Ly1gyCA+PACjZkYILf/QecoJhOs+xj4X79xvnyt5Tlj0ndMhUPHaHnSollWUzyvBJ9rXWv4/UxiVQtFNam1ridWieION4D/CovPFZg01ughdNI+CBM+2lgoaA0BIKZIpTVX9vCvJYy17u4bdX9XIGGdIMo8uE7AXEhEShgOIwowJkfQ5owgEnfxk5w4TfKZvGntLxaYtL11rYGd5JzZGZOKwgLXvwQZrs+gj0A2Uk8gh3ssA/5UKuGjjSWVjHWy9jv8uaee+mpxV2cFDT+6rYxtl/wiKRsSKYDapmuxawIAB0g3b6urlPbLnvIOJQkykvoMV7Q01HCpUKk/dOc0G+7H+u5+PXR9UWP2MQKsxsQ5/7El2B6HLzdPCC2m0XRmmFawieDm7UjP7/juD/r96USnV2pW4okXhgp5/NmHUT1k= vagrant@vagrant)
```
Comment (pluhin) [inline Evgeniya_Pehota/06. Ansible.run/06. Ansible.run.md:71]: Please remove sensitive data from PR

**Review by pluhin — CHANGES_REQUESTED** [review]
Remove whitespace from folder path, or replace by `_`

File: Evgeniya_Pehota/06. Ansible.run/06. Ansible.run.md (hunk 1 above)
Comment (Zhenyakuch) [reply Evgeniya_Pehota/06. Ansible.run/06. Ansible.run.md]: fixed

File: Evgeniya_Pehota/06. Ansible.run/06. Ansible.run.md (hunk 2 above)
Comment (Zhenyakuch) [reply Evgeniya_Pehota/06. Ansible.run/06. Ansible.run.md]: fixed

**Review by pluhin — CHANGES_REQUESTED** [review]
Please remove white-space from names of folders and files

**Pull request closed without merging**